src/frontend/src/common/queries/auth.ts

```diff
@@ -24,6 +24,7 @@ export const USE_USER_INFO = {
   id: "userInfo",
 };
 
+// FIXME (vince): I can't find this used anywhere? If that's true, should delete.
```

Comment:
going back to [where this was added](https://github.com/chanzuckerberg/aspen/pull/475/files), it doesn't seem to ever have been used anywhere. My guess is that fact got lost in the shuffle of the huge PR

Reply:
Thanks for confirming. I've removed it.